.github/workflows/release.yml

```diff
@@ -48,9 +48,9 @@ on:
 jobs:
 
   build-release:
-    runs-on: ubuntu-22.04
-    ## runs-on: ubuntu-latest-devops-xxlarge
-    timeout-minutes: 120
+    ## runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest-devops-xxlarge
+    timeout-minutes: 60
```

Comment:
Yes, on such a powerful runner if builds exceed 60 minutes there must be a non-performance-related problem so good to shorten the timeout.